indexer/indexer.go

```diff
@@ -56,6 +61,17 @@ const (
 
 	// zeroValue is 0 as a string
 	zeroValue = "0"
+
+	// overclockMultiplier is the amount
+	// we multiply runtime.NumCPU by to determine
+	// how many goroutines we should
+	// spwan to handle block data sequencing.
+	overclockMultiplier = 16
```

Comment:
I wonder if there would be a way to make this value a function of the underlying machine's capacity instead of a fixed hard-coded value? seems like a potential risk for running out of threads or memory, but I could be wrong. Is it always the case that 16 routines to one cpu is a good fit?

Reply:
Agree, but we can come back here and improve this afterwards. At the moment just aligning with rosetta-bitcoin